inclusive-design/wecount.inclusivedesign.ca
# Feat: New homepage tiles (resolves #379)

## Description

Update tile icons for home page using new [svgs](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=2422%3A0).

## Steps to test

1. Go to deploy preview
2. Scroll to bottom of homepage
3. Observe cards

**Expected behavior:** 
Cards should be the same as latest [Figma designs](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=2422%3A0)
## Additional information
N/A


## Related issues


closes #379
